### PR TITLE
gemma: Adding new package.

### PR DESCRIPTION
**Pull request by dlkuehn**
Adding a new package : gemma.
Current version           :  0.98.5

Description:  GEMMA is a software toolkit for fast application of linear mixed models (LMMs) and related models to genome-wide association studies (GWAS) and other large-scale data sets.

### CODE REVIEW

**Review by snehring — DISMISSED** [review]
Checksum checks out. Installs with gcc 12.1 on my workstation and my rhel7 build container.

File: var/spack/repos/builtin/packages/gemma/package.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    homepage = "https://github.com/genetics-statistics/GEMMA"
+    url      = "https://github.com/genetics-statistics/GEMMA/archive/refs/tags/v0.98.5.tar.gz"
+
+    maintainers = ['snehring', 'dlkuehn']
+
+    version('0.98.5', sha256='3ed336deee29e370f96ec8f1a240f7b62550e57dcd1694245ce7ec8f42241677')
+
+    depends_on('zlib')
+    depends_on('openblas')
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/gemma/package.py:25]: Only `openblas`? Or can we:
```suggestion
    depends_on('lapack')
```
?

**Review by alalazo — CHANGES_REQUESTED** [review]
One question

File: var/spack/repos/builtin/packages/gemma/package.py (hunk 1 above)
Comment (snehring) [reply var/spack/repos/builtin/packages/gemma/package.py:25]: It has switches in the makefile to maybe make it work with atlas. Other blas/lapack implementations seem to result in linking errors due to specifying -latlas when openblas isn't specified.

Comment (alalazo) [reply var/spack/repos/builtin/packages/gemma/package.py:25]: Thanks, can you either write that in a comment above the `depends_on` or see if we can make it work with other lapack implementations?

Comment (snehring) [reply var/spack/repos/builtin/packages/gemma/package.py:25]: The logic in the makefile is very simple, but it requires that the lapack implementor have literal clblas, lapack, and blas libraries, which many don't. netlib's libraries do, but mkl doesn't for instance.

**Review by alalazo — APPROVED** [review]

**Pull request merged**